spec/support/system_test.rb

```diff
@@ -1,6 +1,11 @@
 RSpec.configure do |config|
   config.before(:suite, type: :system) do
-    Capybara.disable_animation = ".map-selector,.map-selector.show"
+    Capybara.disable_animation = [
+      ".map-selector",
+      ".map-selector.show",
+      ".current-map__token-drawer",
+      ".current-map__token-drawer.show"
+    ].join(",")
```

Comment:
Shall we just go `true` for this and disable all animation?

Reply:
I can check. I thought I had originally tried that and it caused other specs to fail, but it's been long enough now that I'm not certain.

Reply:
Just a thought for the future 👍

Reply:
Yes, I just tried it and remember now. Pointing things out on the map is entirely a css animation and so disabling all animations causes those specs to fail. I've tried changing this to `true` and then on those pointer specs doing

```
around do |example|
    Capybara.disable_animation = false
    example.run
    Capybara.disable_animation = true
  end
```

But that doesn't seem to work when the whole suite is run, and I can't find a combination of a global and a spec-specific thing that works when the whole suite is run. Let me know if you have any ideas, but for now I think I'll leave this as is.